apps/src/p5lab/spritelab/constants.js

```diff
@@ -8,6 +8,7 @@ module.exports.SpritelabReservedWords = [
   'CENTER',
   'World',
   'background',
+  'color',
```

Comment:
i'm not sure i understand how this list woks (mostly focused on this "p5 globals" section) -- are these all keywords that exist in p5[.play] that we want to take control of?

Reply:
these are built-in functions in p5.play that we want to *not* overwrite. This list is then passed into Blockly as reserved words, so that if a student makes a variable named `color`, it generates code like `var color2 = ...` instead of `var color = ...`

Reply:
ahhh, i had it backwards -- thank you!